src/ui/Ethdebugger.js

```diff
@@ -81,7 +81,7 @@ Ethdebugger.prototype.debug = function (tx) {
 
 Ethdebugger.prototype.render = function () {
   var view = yo`<div style=${ui.formatCss(style.font)}>
-        <div style=${ui.formatCss({'max-width': '51%'}, style.innerShift)}>
+        <div style=${ui.formatCss(style.innerShift)}>
```

Comment:
if there's enough width to display the slider, button, etc ... remix will just use it.
Now remix in browser solidity does not use the full width which is a bit strange